filament/backend/include/backend/PixelBufferDescriptor.h

```diff
@@ -247,7 +247,7 @@ class UTILS_PUBLIC PixelBufferDescriptor : public BufferDescriptor {
         }
 
         size_t bpr = bpp * stride;
-        size_t bprAligned = (bpr + (alignment - 1)) & -alignment;
+        size_t bprAligned = (bpr + (alignment - 1)) & (~alignment + 1);
```

Comment:
Is this necessary now? Do the Filament libs build without it in VS2022? If not necessary, I would postpone this (along with other MSVC warning hunt) in [GFX-1590](https://shapr3d.atlassian.net/browse/GFX-1590)

Reply:
Reverted in a17abd4